docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -87,6 +87,40 @@ Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQ
 ----
 
 
+=== Execute sequence of actions
+
+==== *_Info_*
+
+Executes the sequence of web actions
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I execute sequence of actions: $actions
+----
+
+==== *_Parameters_*
+
+. `$actions` - table of actions to execute
```

Comment:
could you please describe the actions? see javadocs or release notes for details

Reply:
described